targets.js

```diff
@@ -76,4 +76,16 @@ const filter = (targets, criteria) => {
   throw new Error ("don't know how to filter ",criteria);
 }
 
-module.exports = {getTargets, filter };
+const shuffle = (targets) => {
+
+  const shuffled = [...targets];
```

Comment:
why are you working on a copy? the result is going to be assigned to the source object anyway

Reply:
Done, works the same